ert_shared/ensemble_evaluator/entity/snapshot.py

```diff
@@ -9,6 +8,8 @@
     get_step_id,
     get_job_id,
 )
+from pyrsistent import freeze, thaw, m
```

Comment:
[Bikeshedding]
I think this is a good example where the namespace/module name makes a lot of sense to keep around and do

```python
import pysistent

...
pyrsistent.m()
```